source/globals.d.ts

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line no-unused-vars
+declare const browser: { [x: string]: any };
+
+type AnyElement = Element | HTMLAnchorElement;
```

Comment:
My plan was to use this, but for whatever reason it still didn't work. My knowledge of TS is near zero.